core/meta/property-descriptor.js

```diff
@@ -370,22 +370,22 @@ exports.PropertyDescriptor = Montage.specialize( /** @lends PropertyDescriptor#
                 deprecate.deprecationWarningOnce("valueDescriptor reference via ObjectDescriptorReference", "direct reference via object syntax");
                 return this._valueDescriptorReference.promise(this.require);
             } else {
-                return Promise.resolve(this._valueDescriptorReference);
+                return this._valueDescriptorReference && Promise.resolve(this._valueDescriptorReference);
```

Comment:
What happens when this._valueDescriptorReference is false? undefined is returned when a promise is expected?

Reply:
Correct. This was a part of Jonathan's commit that made DataMapping synchronous whenever possible.  

https://github.com/montagejs/montage/commit/91ce91e309eb68377e3e8d5a4cb4ca6d4d4476b3

I'm not sure why it's showing up as a change in the merge commit. Oddly, the history of that file does not include the merge commit either.  
https://github.com/montagejs/montage/commits/master/core/meta/property-descriptor.js

Perhaps I missed a step when merging over? The PR said that it could be merged without conflict so I did so without performing any other steps. For instance, I did not merge master into the branch before merging the branch back into master. I don't know if that would make a difference. Do you have an idea?